### DISCUSSION TITLE
[Esame 3 giugno 2024] Esercizio 2

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#455] [Esami M-Z]
![3 giu es.2](https://github.com/user-attachments/assets/6632abf1-f092-42be-999c-e1bf35c6d33b)

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/aa847075-f615-4f3a-b318-06bc439d0d3e)

**Comment by AglaiaNorza** [comment]
![image](https://github.com/user-attachments/assets/e6da6ae3-26d3-4d46-ab21-720e9e9ff862)

**Comment by niccolo-fato** [comment]
<img width="1280" height="1087" alt="image" src="https://github.com/user-attachments/assets/5a23d51e-a52c-4e4c-bd1b-483a3575f93f" />
